internal/db/db.go

```diff
@@ -0,0 +1,50 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+	_ "github.com/lib/pq"
+)
+
+var DB *sql.DB
+
+func InitDB() (*sql.DB, error) {
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatalf("Error loading .env file")
+	}
+
+	connStr := fmt.Sprintf("user=%s dbname=%s password=%s sslmode=disable", GetEnvVariable("DATABASE_USER"), GetEnvVariable("DATABASE_NAME"), GetEnvVariable("DATABASE_PASSWORD"))
+
+	DB, err = sql.Open("postgres", connStr)
+	if err != nil {
+		return nil, fmt.Errorf("error opening database connection: %v", err)
+	}
+
+	err = DB.Ping()
+	if err != nil {
+		return nil, fmt.Errorf("error pinging database: %v", err)
+	}
+
+	return DB, nil
+}
+
+func GetEnvVariable(key string) string {
```

Comment:
Keep in mind that variables and function named with a capital letter are exported from the package (meaning they're public) and the rest are not (they're private). So keep everything private, except for the functions and constants intended to be used by other packages.